iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/AffMatchingJob.java

```diff
@@ -112,12 +118,31 @@ private static AffMatchingService createAffMatchingService(JavaSparkContext spar
         AffOrgMatcher firstWordsHashBucketMatcher = createFirstWordsHashBucketMatcher();
         
         
+        affMatchingService.setAffOrgMatchers(of(docOrgRelationMatcher, mainSectionHashBucketMatcher, firstWordsHashBucketMatcher));
         
         
-        affMatchingService.setAffOrgMatchers(of(docOrgRelationMatcher, mainSectionHashBucketMatcher, firstWordsHashBucketMatcher));
+        AffMatchOrganizationAltNameFiller altNameFiller = createAffMatchOrganizationAltNameFiller();
+        affMatchingService.setAffMatchOrganizationAltNameFiller(altNameFiller);
         
         return affMatchingService;
     }
     
+    private static AffMatchOrganizationAltNameFiller createAffMatchOrganizationAltNameFiller() throws IOException {
+        
+        AffMatchOrganizationAltNameFiller altNameFiller = new AffMatchOrganizationAltNameFiller();
```

Comment:
Indeed, they are very similar. I can't think of any better name though.
If you have some proposition. Please tell.